app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -55,21 +57,21 @@ def run(args = {})
     object = object_type.constantize.find_by(:id => object_id) if object_type && object_id
     object.before_ae_starts({}) if object.present? && object.respond_to?(:before_ae_starts)
 
-    creds = credentials&.transform_values do |val|
```

Comment:
will this be flat or do we need to be recursive?

Not sure if I like hashes or $ better. Feels like we are mixing paradigms. The `.$` will use path and stuff, vs with our hash, we are going the other way.
(I'd prefer just going .$ and $, but I can see why we are going the hash route)